clustered/common/src/main/java/org/ehcache/clustered/common/internal/store/Util.java

```diff
@@ -156,6 +158,16 @@ public static Object unmarshall(byte[] payload) {
     }
   }
 
+  public static Object unmarshall(ByteBuffer payload) {
```

Comment:
This method is not being used anywhere. This class has findbugs failures as well